contrib/cmus-autoupdate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+usage() {
+    printf "%s <music directory>
+    Automatically add/remove files to/from the Library
+    when files in the given directory are changed." $0
+    exit
+}
+
+[ -z "$1" ] && usage
+
+which inotifywait >/dev/null || { printf "This script requires inotify-tools to be installed."; exit; }
+
+# NB. if your files are on removable storage, -e unmount may be useful here.
+inotifywait --csv --monitor --recursive -e create -e delete "$1" | \
```

Comment:
This is racy; created files may not be finished writing. The `close_write` may be a better alternative. Also, `moved` is also required to catch all relevant changes.